aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -738,6 +738,19 @@ void lcm_kernel(TensorIterator& iter) {
     });
 }
 
+void hypot_kernel(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES(iter.dtype(), "hypot_cpu", [&]() {
```

Comment:
The PR adds a vectorized version of bfloat16 hypot but this dispatch would only be for float32 and float64, right?